### DISCUSSION TITLE
How to map 'note(s)' to 'description' with FIDI?

### DISCUSSION THREAD

**Discussion by jorgnyg** [firefly-iii/firefly-iii#5936] [Firefly III] [answered]
Using FIDI with Nordigen and my bank exports nothing for the 'description' field, making the data in Firefly appearing as (no description). The correct descriptions seems to be in the notes.

Is it possible to somehow map the notes to description, so that the description field has the correct value?
One workaround would be to import data from Nordigen with no data in description, export as CSV, create a script to replace (no description) with the content of note and finally import the modified CSV with FIDI.

**Comment by JC5** [comment, marked as answer]
This is something you may be able to raise with Nordigen, at support@nordigen.com. They are supposed to map uniformly, so each bank shows the same structure. Tell them the bank you use and show them an example from the FIDI logs. They're usually quick to respond and will follow up.

**Reply by jorgnyg** [reply]
Great! Will send a mail right away.
Thanks.

**Reply by jorgnyg** [reply]
@JC5 Well, they said "The data that is offered differs from bank to bank". So seems like they're meaning it's working as expected.
Is it a way to map roles with a json config file when importing from Nordigen?

**Reply by JC5** [reply]
I'm afraid not, Nordigen is always imported as-is.
